docs/services/wordpress.md

```diff
@@ -0,0 +1,76 @@
+# Wordpress
+
+[WordPress](https://wordpress.org/) is a widley used open source web content management system that this playbook can install, powered by the [mother-of-all-self-hosting/ansible-role-wordpress](https://github.com/mother-of-all-self-hosting/ansible-role-wordpress) Ansible role.
+
+## Dependencies
+
+This service requires the following other services:
+
+- a [MariaDB](mariadb.md) database
+- a [Traefik](traefik.md) reverse-proxy server
+
+## Configuration
+
+```yaml
+########################################################################
+#                                                                      #
+# wordpress                                                            #
+#                                                                      #
+########################################################################
+
+wordpress_enabled: true
+
+wordpress_hostname: example.org
+
+########################################################################
+#                                                                      #
+# /wordpress                                                           #
+#                                                                      #
+########################################################################
+```
+
+### Enable MariaDB as database
+
+If not already done you need to enable MariaDB so Wordpress can use it as database.
+
+```yaml
+########################################################################
+#                                                                      #
+# mariadb                                                              #
+#                                                                      #
+########################################################################
+
+mariadb_enabled: true
+
+# Put a strong password below, generated with `pwgen -s 64 1` or in another way
+mariadb_root_passsword: ''
+
+########################################################################
+#                                                                      #
+# /mariadb                                                             #
+#                                                                      #
+########################################################################
+```
+
+
```

Comment:
Isn't this unnecessary? We already list and link to MariaDB (`mariadb.md`) in the **Dependencies** section, so duplicating MariaDB's installation instructions here seems redundant - we don't do this for other services.

Reply:
True, fixed